torchdynamo/__init__.py

```diff
@@ -16,6 +17,9 @@
 from .utils import guard_failures
 from .utils import orig_code_map
 
+functorch.compile.config.use_fake_tensor = True
```

Comment:
I remember it was causing some cyclic dependencies - https://github.com/pytorch/torchdynamo/pull/1096